pulp_smash/tests/platform/cli/test_content_sources.py

```diff
@@ -0,0 +1,97 @@
+# coding=utf-8
+"""Tests for list/refresh/delete content sources."""
+from __future__ import unicode_literals
+
+import unittest2
+from packaging.version import Version
+
+from pulp_smash import cli, config
+from pulp_smash.constants import CONTENT_SOURCE_ID
+
+
+def _get_content_source_ids(server_config):
+    """Get the id list of all content sources, or None if the list is empty.
+
+    :param server_config: pulp_smash.config.ServerConfig server_config:
```

Comment:
This line has an extra `:`. Please read http://www.sphinx-doc.org/en/stable/domains.html#the-python-domain, and please verify the output of `make docs-html`.